CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+Please see the [Material Components code of conduct](https://github.com/material-components/material-components/blob/develop/CODE_OF_CONDUCT.md).
```

Comment:
Should the language here be more along the lines of

'The MC COC applies to this repository'?

If this is our standard language for sharing licenses, codes of conduct, etc, then this is fine with me.

Reply:
This is the standard language. Matches [https://github.com/material-components/material-components-ios/blob/develop/CODE_OF_CONDUCT.md](https://github.com/material-components/material-components-ios/blob/develop/CODE_OF_CONDUCT.md)

Reply:
Sounds good to me